manifests/service.pp

```diff
@@ -14,4 +18,11 @@
       File["${cfg_dir}/zoo.cfg"]
     ]
   }
+
+  if $restart_zookeeper {
```

Comment:
What about the package?

Sometimes what people do is move this to the init class and do
```
if $restart {
  Class['zookeeper::install'] ~> Class['zookeeper::service']
  Class['zookeeper::config'] ~> Class['zookeeper::service']
}
```

This way you can't "forget" to restart if *anything* changes inside the config class, instead of whitelisting files that might be important.

Reply:
Good point. However, for this change (rspec fixes), I would prefer to keep status quo and update that in a separate commit.